MIGRATION.md

```diff
@@ -1,5 +1,11 @@
 # Upgrading from 7.x to 8.x
 
+## Removal of the `tracingOrigins` option
```

Comment:
This likely needs to be rewritten since we're removing `BrowserTracing` completely anyway but for now it's okay. Was just worried because we didn't mention that we removed `tracePropaagationTargets` in the `BrowserTracing` options but realized it's not gonna matter 😅